backend/src/dao/WorkspaceDAO.class.php

```diff
@@ -174,11 +174,17 @@ public function storeFileMeta(int $workspaceId, File $file): void {
     }
 
 
+    public function getFileNames(int $workspaceId): array {
+
+        return $this->_('select type, name from files where workspace_id = ?', [$workspaceId], true);
+    }
+
+
     /**
      * @codeCoverageIgnore
      */
-    public function deleteFileMeta(int $workspaceId, $name): void {
+    public function deleteFileMeta(int $workspaceId, string $name, string $type): void {
```

Comment:
Erschließt sich vielleicht aus dem Kontext, aber mir ist nicht klar was "meta" ist und deswegen ist der Methodenname iwie unklar. Könnte man vllt verbessern.

Reply:
Tja, die in der Datenbank abgelegten Metadaten zu den Dateien. Wie sollte man es anders nennen? Werd mal drüber nachdenken.